docs/src/components/ComponentDoc/ComponentExample/ComponentExample.tsx

```diff
@@ -724,10 +724,10 @@ class ComponentExample extends React.Component<ComponentExampleProps, ComponentE
   }
 }
 
-const ComponentExampleWithTheme = React.forwardRef((props: ComponentExampleProps) => (
+const ComponentExampleWithTheme = props => (
```

Comment:
`React.forwardRef` doesn't make a sense because the second `ref` argument is not used.

Reply:
👍